lib/utils/filename-normaliser.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const path = require('path');
+const findup = require('find-up');
+const { existsSync } = require('fs');
+
+const pkg = require(findup.sync('package.json'));
+
+const normaliseFileName = (fileName = '') => {
+  // not a valid file name
+  if (path.basename(fileName) === fileName) {
```

Comment:
I don't understand. Why is this invalid? The comment could probably use an example 🤔

Reply:
Added some examples.